readthedocs/core/static-src/core/js/doc-embed/mkdocs.js

```diff
@@ -27,7 +27,9 @@ function init() {
           value: 'file'
       }).appendTo('#rtd-search-form');
 
-      $("#rtd-search-form").prop("action", rtd.api_host + "/search/");
+      var search_url = window.location.protocol + "//" + window.location.hostname +
+                       "/" + rtd.language + "/" + rtd.version + "/search.html";
```

Comment:
This doesn't respect subprojects and aliases. We can't assume the project path is `https://hostname/language/version/search.html`, it could be `https://hostname/projects/subproject/language/version/search.html`. Same for aliases here.